### PR TITLE
[Replace Non-Blocking Usage] LPS-198502 Replace non-blocking ServiceProxyFactory usages in classes ServiceAccessPolicyManagerUtil and TunnelAuthenticationManagerUtil

### PR DESCRIPTION
**Pull request by W2jx**
resend, fixed duplicate spring bean entries.

Previous: https://github.com/liferay-core-infra/liferay-portal/pull/5939

### CODE REVIEW

**Comment by W2jx** [comment]
ci:test:relevant

**Comment by W2jx** [comment]
ci:stop

**Comment by W2jx** [comment]
no conflicts(11/6 9:12)

**Comment by W2jx** [comment]
ci:test:relevant

**Comment by zhangyan0701** [comment]
ci:test:relevant

**Comment by zhangyan0701** [comment]
ci:test:upgrade

**Comment by W2jx** [comment]
no conflicts(11/7 9:34)

File: portal-impl/src/META-INF/security-spring.xml [hunk 1]
```diff
@@ -20,6 +20,7 @@
 	<bean class="com.liferay.portal.kernel.security.auth.DefaultFullNameValidator" id="com.liferay.portal.kernel.security.auth.DefaultFullNameValidator" />
 	<bean class="com.liferay.portal.kernel.security.auth.DefaultScreenNameGenerator" id="com.liferay.portal.kernel.security.auth.DefaultScreenNameGenerator" />
 	<bean class="com.liferay.portal.kernel.security.auth.DefaultScreenNameValidator" id="com.liferay.portal.kernel.security.auth.DefaultScreenNameValidator" />
+	<bean class="com.liferay.portal.security.auth.tunnel.TunnelAuthenticationManagerImpl" id="com.liferay.portal.kernel.security.auth.tunnel.TunnelAuthenticationManager" />
```
Comment (dantewang) [inline portal-impl/src/META-INF/security-spring.xml:23]: @W2jx It seems the beans are sorted based on their classes? It won't work if we keep it at line 86?

**Review by dantewang — CHANGES_REQUESTED** [review]

File: portal-impl/src/META-INF/security-spring.xml (hunk 1 above)
Comment (W2jx) [reply portal-impl/src/META-INF/security-spring.xml:23]: Hi @dantewang, SF asks us to sort these beans based on bean id.

Comment (W2jx) [reply portal-impl/src/META-INF/security-spring.xml:23]: Hi @dantewang , yes, it will work, the id can be any string, as long as it is unique

Comment (dantewang) [reply portal-impl/src/META-INF/security-spring.xml:23]: @W2jx So the order in the xml file doesn't matter. Let's keep the original id, and let the util use it.

Comment (W2jx) [reply portal-impl/src/META-INF/security-spring.xml:23]: Ok, I will resend it.

**Comment by W2jx** [comment]
ci:test:sf

**Comment by W2jx** [comment]
ci:test:relevant

**Comment by W2jx** [comment]
ci:test:upgrade

**Comment by W2jx** [comment]
ci:test:startup

**Comment by W2jx** [comment]
no conflicts(11/8 10:05)

**Comment by dantewang** [comment]
ci:forward:force

**Pull request closed without merging**